bower.json

```diff
@@ -18,7 +18,7 @@
     "angular-sanitize": "~1.5.8",
     "angular-scroll-glue": "~2.0.7",
     "angular-ui-router": "~0.3.1",
-    "angular-ui-tree": "2.22.5",
+    "angular-ui-tree": "https://github.com/FinnStutzenstein/angular-ui-tree.git#cd42228fcd013295a43374ec15553e7984bd29bc",
```

Comment:
This has to be explained. :-)

Reply:
OK ;)
I merged [one PR](https://github.com/angular-ui-tree/angular-ui-tree/pull/934) from the upstream into my fork, because the ui-tree is not maintained regularly. This PR adds some sensitivity for horizontal movement of nodes across heavy nested trees. This improves the handling very much. I'll keep track whether this PR is going to be merged in the upstream.

Reply:
Ok.